ral-facilities/datagateway
# Restoring DataGateway homepage to master branch #775

## Description
As per the issue, DataGateway's homepage that had been merged into master on May 21st was accidentally removed on May 24th and only now has the problem been noticed. The main purpose of this PR is to restore that homepage back to master. The only change that has been made is the addition of some template string literals where they could have been used to improve the code style. This was done in https://github.com/ral-facilities/datagateway/commit/dfa8ccf01b68e53f33ba247ade31ea5c70938294

Bear in mind that this work was already approved in https://github.com/ral-facilities/datagateway/pull/662/ however you are free to review and test it again if you wish.

## Testing instructions
Test this with SciGateway. To load the homepage on SciGateway, ensure your datagateway-dataview-settings.json is up to date with changes made in this PR. This adds the homepage as a new route and should appear as below:

```
"section": "Homepage",
 "link": "/datagateway",
 "displayName": "DataGateway",
 "order": 0
```

Now add the following to SciGateway's public/settings.json:

```
"homepageUrl": "/datagateway"
```

This will ensure all navigations to / go to /datagateway.
When you first load up SciGateway, the new homepage should load first. Any subsequent attempts should also go to the new homepage and clicking the SciGateway logo on the navigation bar at the top of the screen should also redirect to the new homepage.

- [x] Check the homepage is always redirected to if trying to access /

## Agile board tracking
Connect to #431
Closes #775
